tyr/servers/iis/node.py

```diff
@@ -69,31 +69,27 @@ def __init__(self, group=None, server_type=None, instance_type=None,
         ]
 
         if self.mongos_service:
-            mongo_ops = ("MongosHost", "Disabled", "MongosService")
+            mongo_ops = ("mongos_host", "no_mongos", "mongos_service")
             if self.mongos_service not in mongo_ops:
                 raise ValueError(
                     "Mongo service name must be one of: {0}".format(
                         mongo_ops))
 
         self.ports_to_authorize = [9000, 9001, 8095, 8096]
-
-        self.IAM_ROLE_POLICIES.append('allow-web-initialization-{environment}')
-        self.IAM_ROLE_POLICIES.append('allow-outpost-sns-{environment}')
+        self.IAM_MANAGED_POLICIES.append('hudl-webserver-{environment}-multiverse')
 
     def configure(self):
         super(IISNode, self).establish_logger()
         super(IISNode, self).configure()
 
     @property
     def user_data(self):
-        data = {
-            "bucket": "hudl-config",
-            "key": "{env}-mv-web/init.config.json".format(
-                env=self.environment[0]),
-            "mongos": self.mongos_service,
-            "mongoServers": self.mongo_servers
-        }
-
-        ud = json.dumps(data)
-        self.log.info("Setting user data to: {ud}".format(ud=ud))
-        return ud
+        # read in userdata file
+        user_data = None
+        try:
```

Comment:
I think we should handle this better - if we can't find the userdata script for some reason, we should probably exit before creating the LC, ASG, and the subsequent servers as it's a vital aspect of it.